src/utils/loadFiles.ts

```diff
@@ -0,0 +1,24 @@
+import { readdir } from "fs/promises";
+import type { Collection } from "discord.js";
+import type { DiscordEvent } from "../types/DiscordEvent";
+import type { Command } from "../types/Command";
+
+async function loadFiles<T extends Command | DiscordEvent>(collection: Collection<string, T>, path = "./commands"): Promise<void> {
```

Comment:
is there really a need for default argument?

Reply:
nope, will be removed.